scripts/build-json/build-json.js

```diff
@@ -17,26 +18,36 @@ function walk(directory, filepaths) {
     }
 }
 
-function collectItems(directory, searchPaths) {
+function findItems(directory, searchPaths) {
     let filepaths = [];
     walk(directory, filepaths);
-    filepaths = filepaths.map( 
-        filepath => filepath.slice(path.join(process.cwd(), './content/').length) 
-    ).filter(filepath => !searchPaths.length || searchPaths.some(searchPath => filepath.includes(searchPath)))
-    return filepaths;
+    return filepaths.filter(
+        filePath => !searchPaths.length || searchPaths.some(searchPath => filePath.includes(searchPath)));
 }
 
 function buildJSON(searchPaths) {
     let errors = 0;
-    const items = collectItems(path.resolve(process.cwd(), './content'), searchPaths);
+    const items = findItems(path.resolve(ROOT, 'content'), searchPaths);
     if (!items.length && searchPaths.length) {
         console.error("No elements found");
         errors++;
     }
-    for (let item of items) {
-        const parsed = path.parse(item);
-        errors += buildPage.buildPageJSON(parsed.dir, parsed.base);
-    }
+    
+    items.forEach(async item => {
+        let built
+        try {
+            built = await buildPage.buildPageJSON(item);
+            const { docsPath, destPath } = built;
+            if (destPath !== null) {
+                console.log(`Packaged ${docsPath} to ${destPath}`);
```

Comment:
I think this console output is harder to read than the previous version: the lines are much longer and very repetitive.